pyiron_atomistics/sphinx/base.py

```diff
@@ -1592,7 +1607,12 @@ def check_setup(self):
                 else:
                     warnings.warn(
                         "Number of empty states was not specified. Default: "
-                        + "3+NIONS for non-magnetic systems"
+                        + "3+NIONS*0.5 for non-magnetic systems"
+                    )
+            else:
+                if self.input["EmptyStates"] < 0.5 * int(len(self.structure) + 3):
```

Comment:
Isn't this normal for semi conductors? @skatnagallu @freyso

Reply:
For semiconductors this doesnt matter right? You shouldn't set it for semiconductors. And then SPHInX defaults to 0.